tools/metadataSorter.ts

```diff
@@ -0,0 +1,60 @@
+import "source-map-support/register.js";
+
+import { readFile, writeFile } from "fs/promises";
+import glob from "glob";
+
+import { Metadata } from "./classes/PresenceCompiler.js";
+import getLatestSchema from "./util/getLatestSchema.js";
+
+const { url: latestSchema } = await getLatestSchema(),
+	presences = glob.sync("websites/*/*/metadata.json");
+
+for (const presence of presences) {
+	const {
+		altnames,
+		author,
+		category,
+		color,
+		contributors,
+		description,
+		iframe,
+		iFrameRegExp,
+		logo,
+		readLogs,
+		regExp,
+		service,
+		settings,
+		thumbnail,
+		tags,
+		url,
+		version,
+	} = JSON.parse(await readFile(presence, "utf8")) as Partial<Metadata>;
```

Comment:
why not just ✨require✨

Reply:
Updated with the change of the needed createRequire function as ESM does not natively have it and this way is the recommended way.

Reply:
....